packages/wrangler/src/__tests__/rollback.test.ts

```diff
@@ -119,7 +119,7 @@ describe("rollback", () => {
 		mockPostDeployment();
 
 		mockPrompt({
-			text: "Please provide a message for this rollback (120 characters max, optional)?",
+			text: "Please provide an optional message for this rollback (120 characters max)?",
```

Comment:
this failed locally since we changed this messaging during review @CarmenPopoviciu but CI was passing?
Is CI not running this test for some reason or any idea why this passed?